samples/drumthumper/src/main/cpp/DrumPlayerJNI.cpp

```diff
@@ -47,7 +47,7 @@ static SimpleMultiPlayer sDTPlayer;
 JNIEXPORT void JNICALL Java_com_plausiblesoftware_drumthumper_DrumPlayer_setupAudioStreamNative(
         JNIEnv* env, jobject, jint numChannels) {
     __android_log_print(ANDROID_LOG_INFO, TAG, "%s", "init()");
-    sDTPlayer.setupAudioStream(numChannels);
+    sDTPlayer.setupAudioStream(numChannels, oboe::PerformanceMode::LowLatency);
```

Comment:
Before this change, the performance mode is not set, it is by default as `None`. If we want to change to use LowLatency mode, I'd suggest to do it in other CL and keep the previous behavior here.

Reply:
Done